src/lib/jwtNode.ts

```diff
@@ -1,24 +1,168 @@
-import jwt from 'jsonwebtoken';
+import { createPrivateKey } from 'crypto';
```

Comment:
I wonder if there's a way to avoid using this from node crypto... maybe jose has an alternative? jose is browser compatible, node crypto is not...